release/cibuild.py

```diff
@@ -382,6 +384,7 @@ def build_wininstaller(be: BuildEnviron) -> None:  # pragma: no cover
     click.echo("Building wininstaller package...")
 
     IB_VERSION = "20.12.0"
+    IB_SETUP_SHA256 = "657f4785c7d70f140468435b99e79ced813e7e051106e7525e0c819efffb40d3"
```

Comment:
We now also verify the hash for InstallBuilder, which we use to build windows installers. :)